lib/playback.test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { withPlayback } = require('../index.js');
+
+const options = {
+    dev: true,
+    isServer: true,
+};
+
+const mockFunction = jest.fn();
```

Comment:
This seems strange to have it here. I would move it to the place where is needed.